runtime/src/epoch_stakes.rs

```diff
@@ -5,14 +5,17 @@ use {
     solana_clock::Epoch,
     solana_pubkey::Pubkey,
     solana_vote::vote_account::VoteAccountsHashMap,
-    std::{collections::HashMap, sync::Arc},
+    std::{
+        collections::HashMap,
+        sync::{Arc, OnceLock},
+    },
 };
 
 pub type NodeIdToVoteAccounts = HashMap<Pubkey, NodeVoteAccounts>;
 pub type EpochAuthorizedVoters = HashMap<Pubkey, Pubkey>;
 pub type SortedPubkeys = Vec<(Pubkey, BLSPubkey)>;
 
-#[derive(Clone, Debug, Serialize, Deserialize, Default)]
+#[derive(Clone, Debug, Default)]
 #[cfg_attr(feature = "frozen-abi", derive(AbiExample))]
```

Comment:
we can delete this now, right?

Reply:
Unfortunately no. The abi package is not smart enough to figure out we don't need this any more because of OnceLock, so it won't compile. But we won't use it.
See my previous attempt:
https://buildkite.com/anza/alpenglow/builds/1761#0198f781-2060-4f10-9cf7-9b934a1b11c6